LinkResolverService.cs

```diff
@@ -5,27 +5,28 @@
 using EPiServer.Web;
 using EPiServer.Web.Routing;
 using System;
+using System.Globalization;
 
 namespace Blend.Optimizely
 {
     [ServiceConfiguration]
     public class LinkResolverService
     {
-
         private Injected<IContentLoader> ContentLoader { get; }
+
         private Injected<ISiteDefinitionResolver> SiteDefinitionResolver { get; }
-        private Injected<IUrlResolver> UrlResolver { get; }
 
+        private Injected<IUrlResolver> UrlResolver { get; }
 
-        public virtual ResolvedLink? TryResolveLink(object? contentLink, LinkOptions options = LinkOptions.None)
+        public virtual ResolvedLink? TryResolveLink(object? contentLink, LinkOptions options = LinkOptions.None, string languageBranchId = "")
         {
             if (contentLink is null)
                 return null;
 
             var resolvedLink = contentLink switch
             {
                 IResolvable resolvable => resolvable.Resolve(options),
-                IContent content => ResolveIContent(content, options),
+                IContent content => ResolveIContent(content, languageBranchId, options),
                 Url url => ResolveUrl(url, options),
                 LinkItem linkItem => ResolveLinkItem(linkItem, options),
                 ContentReference contentReference => ResolveContentReference(contentReference, options),
```

Comment:
Is there a reason we don't pass the language ID here and in the branch below? There is a method to resolve a content reference with a language identifier.